github/ci/prow-deploy/files/jobs/kubevirt/applications-aware-quota/applications-aware-quota-postsubmits.yaml

```diff
@@ -0,0 +1,196 @@
+postsubmits:
+  kubevirt/applications-aware-quota:
+  - name: push-release-aaq-images
+    branches:
+    - release-v\d+\.\d+
+    cluster: kubevirt-prow-control-plane
+    always_run: true
+    optional: false
+    skip_report: true
+    annotations:
+      testgrid-create-test-group: "false"
+    decorate: true
+    decoration_config:
+      timeout: 1h
+      grace_period: 5m
+    labels:
+      preset-podman-in-container-enabled: "true"
+      preset-docker-mirror-proxy: "false"
```

Comment:
Removing the preset here is better than setting it to false - I think this is repeated throughout the jobs.

Reply:
Any reason in particular why you are not using the image mirror?

Reply:
Done.